### PR TITLE
Upgrade admin and reactions bugfixes

### PR DESCRIPTION
**Pull request by ashwiniHerle**
Pull request to fix minor issues in admin and reaction components:

- Reduced font size where necessary
- Removed all unused legacy imports (For all components where applicable)
- Minor spacing and styling fixes
- Changed css color classes to use bootstrap utility classes

### CODE REVIEW

File: app/packs/src/apps/admin/DelayedJobs.js [hunk 1]
```diff
@@ -52,14 +51,14 @@ export default class DelayedJobs extends Component {
...
-        <th className="w-50 fs-4">Last Errors</th>
+        <th className="w-4" colSpan="2">ID</th>
+        <th className="w-5">Queue</th>
+        <th className="w-5">Job Class</th>
+        <th className="w-5">Run At</th>
+        <th className="w-5">Failed At</th>
+        <th className="w-4">Attempts</th>
+        <th className="w-4">Priority</th>
+        <th className="w-50">Last Errors</th>
       </tr>
```
Comment (beque) [inline app/packs/src/apps/admin/DelayedJobs.js:62]: The same here. w-50 exists
Just take a look how the table looks like without w-4 and w-5

File: app/packs/src/apps/admin/AdminGroupElement.js [hunk 2]
```diff
@@ -175,12 +174,12 @@ export default class AdminGroupElement extends React.Component {
...
-                          <td className="w-20 py-3 fs-4">{ }</td>
-                          <td className="w-15 py-3 fs-4">{ }</td>
-                          <td className="w-30 py-3 fs-4">
+                          <td className="w-5  py-3">{i + 1}</td>
+                          <td className="w-20 py-3">{u.name}</td>
+                          <td className="w-10 py-3">{u.initials}</td>
+                          <td className="w-20 py-3">{ }</td>
+                          <td className="w-15 py-3">{ }</td>
+                          <td className="w-30 py-3">
                             <DeleteGroupDeviceButton rootType={'Group'}
```
Comment (beque) [inline app/packs/src/apps/admin/AdminGroupElement.js:183]: You can remove the w-5, w-20, w-10, w-15 styles.
They do not exist in bootstrap.
Everything should look the same

**Review by beque — COMMENTED** [review]
Every thing looks fine.
Just take a look at the w- classes for tables.
Then you can merge

File: app/packs/src/apps/admin/AdminGroupElement.js (hunk 2 above)
Comment (ashwiniHerle) [reply app/packs/src/apps/admin/AdminGroupElement.js:183]: Yes it was indeed redundant. Made necessary changes.

**Pull request merged**